context_test.go

```diff
@@ -41,4 +51,48 @@ var _ = Describe("Context", func() {
 			Expect(ok).To(BeFalse())
 		})
 	})
+
+	Context("Not yet implemented", func() {
```

Comment:
it's on purpose because the tested functions do nothing and just fullfil the interface